src/blocks/icon-list/edit.js

```diff
@@ -273,6 +280,22 @@ class KadenceIconLists extends Component {
 			this.onMove( oldIndex, oldIndex - 1 );
 		};
 	}
+
+	onMoveLeft(index) {
+		return () => {
+			if(this.props.attributes.items[index].level > 0)
```

Comment:
What happens after plugin update when level property isn't defined ?
Should null-checks be added at this place ?

Reply:
Yeah, I've added some lodash `get()` checks [in this commit](https://github.com/kadencewp/kadence-blocks/pull/167/commits/84e18ab179e458b7b4c9149c8d5223f5d5cc79f2#diff-1db572eb50221c61511c1b23cdd04e39fffb47c247bace5f8b468e9dab029b76). The level property is undefined for existing icon lists